modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -162,11 +156,11 @@ a|Optional: The URL of the Ignition config for the live boot. For example, this
 [discrete]
 == `coreos-installer` options for ISO install
 
-You can run the `coreos-installer` command to identify various artifacts to include, to work with disk partitions, and to set up networking. In some cases, you can configure features on the live system and copy them to the installed system.
+The `coreos-installer` command is another way for you to add boot options to the {op-system} live installer.
```

Comment:
`add boot options to the live installer` doesn't make sense here.  Instead of this paragraph and the next, maybe something like:

You can also install {op-system} by invoking the `coreos-installer` command directly from the command line.  The kernel arguments in the previous table provide a shortcut for automatically invoking `coreos-installer` at boot time, but you can pass similar arguments directly to `coreos-installer` when running it from a shell prompt.

Reply:
I like this, resolved.